app/app.html

```diff
@@ -2,9 +2,15 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <meta http-equiv="Content-Security-Policy" content="
+      default-src 'self';
+      connect-src 'self' http://localhost:* ws://localhost:* https://api.coinmarketcap.com http://zap.jackmallers.com https://testnet-api.smartbit.com.au;
```

Comment:
https://zap.jackmallers.com/?

Reply:
Currently, we pull the list of suggested nodes from https://zap.jackmallers.com/suggested-peers

Reply:
How about using http instead of https in the CSP?

Reply:
When webpack devserver runs, it runs over http, so all the calls to localhost go over http. It is possible to tell webpack devserver to run using https however when you do that it generates and uses a self signed certificate which electron doesn't know about or trust - so electron will not run correctly as it warns about the unknown/invalid certificate

Reply:
I mean specifically for accessing `zap.jackmallers.com`. Agree `http://localhost` is fine.

Reply:
Ahh yes. I had that change on local already but wasn't included in what I pushed for some reason. Have updated now.